src/seedu/addressbook/AddressBook.java

```diff
@@ -554,11 +561,43 @@ private static boolean isDisplayIndexValidForLastPersonListingView(int index) {
      * @param deletedPerson successfully deleted
      * @return successful delete person feedback message
      */
-    private static String getMessageForSuccessfulDelete(String[] deletedPerson) {
+    private static String getMessageForSuccessfulDelete(HashMap<PersonProperty,String> deletedPerson) {
         return String.format(MESSAGE_DELETE_PERSON_SUCCESS, getMessageForFormattedPersonData(deletedPerson));
     }
 
     /**
+     * Sort and display list of all persons in alphabetical order
```

Comment:
the comment should be in active verb form. 
`Sorts and displays the list of people in alphabetical order` may be more suitable here.